docs/operator-manual/declarative-setup.md

```diff
@@ -682,6 +682,3 @@ patchesStrategicMerge:
 
 The live example of self managed Argo CD config is available at [https://cd.apps.argoproj.io](https://cd.apps.argoproj.io) and with configuration
 stored at [argoproj/argoproj-deployments](https://github.com/argoproj/argoproj-deployments/tree/master/argocd).
-
-!!! note
-    You will need to sign-in using your GitHub account to get access to [https://cd.apps.argoproj.io](https://cd.apps.argoproj.io)
```

Comment:
Why is this removed?

Reply:
The URL seems to be no longer valid

Reply:
Yeah and that's something I had on my fork since Feb 22 and haven't pushed it - hence it made this PR "accidentaly"

Reply:
I think @leoluz is planning to fixcd.apps.argoproj.io soon.

Reply:
Is there a task for this so I can link it here?

Reply:
Just added one: https://github.com/argoproj/argo-cd/issues/9185